DuckDuckGo/AppUserDefaults.swift

```diff
@@ -141,6 +143,14 @@ public class AppUserDefaults: AppSettings {
 }
 
 extension AppUserDefaults: AppConfigurationFetchStatistics {
+    var lastConfigurationRefreshDate: Date? {
```

Comment:
We could probably move to @UserDefaultsWrapper and use distantPast as a default

Reply:
Great suggestion, thanks @bwaresiak – will update.